src/main/java/ch/uzh/ifi/hase/soprafs21/controller/UserController.java

```diff
@@ -48,11 +48,7 @@ public UserGetDTO createUser(@RequestBody UserPostDTO userPostDTO) {
     @ResponseStatus(HttpStatus.OK)
     @ResponseBody
     public String login(@RequestBody UserPostDTO userPostDTO){
-
-        throw new UnsupportedOperationException("Not implemented yet");
-
-        // checks User input and returns token when successful
-        //return userService.loginUser(userPostDTO);
+        return userService.loginUser(userPostDTO);
```

Comment:
⚠️ userPostDTO needs to be converted to internal representation (namely type `User`) before passing it to the user service. (Via DTOMapper)